manifests/cf-manifest/runtime-config/datadog-addon.yml

```diff
@@ -0,0 +1,6 @@
+addons:
+  - name: datadog-agent
```

Comment:
The case I was describing was this one: https://github.com/geofffranks/spruce/issues/171 

But I agree that the issue should be solved by whoever encounters the problem.